packages/client/src/client.d.ts

```diff
@@ -46,5 +46,8 @@ declare class Client {
   request(data: ClientRequest, cb?: (err: ResponseError, response: [ClientResponse, any]) => void): Promise<[ClientResponse, any]>;
 }
 
-declare const client: Client & { Client: typeof Client };
+declare const client: Client;
+// @ts-ignore
 export = client
+
+export {Client};
```

Comment:
We would need to revert this change also if we move back to `declare const client: Client & { Client: typeof Client };`

Reply:
left a comment above about why I did this. let me know what you think